### ISSUE TITLE
LABS version is almost useless

### ISSUE THREAD

**Issue by Sipke82** [krzychu124/Cities-Skylines-Traffic-Manager-President-Edition#380]
I quite like the LABS - STABLE construct. However, as the current release frequency is low, I think it does not make much sense. Because most of the times STABLE and LABS are both same version. 
In the rare cases that LABS is more updated then STABLE, it contains a bugfix which we would like to have in STABLE as well, instead of some " awesome experimental features", which would be the intended purpose IMO. 
And finally, some steam users find the two different versions confusing and comments and discussions are split over 2 workshop pages.

Therefore i think there are 2 options:
- increase the amount of releases of the LABS version (preferred option)
- or go back to one version (this would free up some resources as we no longer have to maintain 2 versions)

**Comment by aubergine10** [comment]
LABS will start to become more important once the Harmony migration (TMPE v11.0) is done.

The goal is that all new updates will be tested on LABS version first, and only later moved to STABLE once we know they are... stable.

Currently both versions have same mod because both were updated for Campus update.

**Comment by FireController1847** [comment]
I strongly disagree with this. Like aubergine said, LABS is used to test early version features and things before they are ready for true development. By your same logic, Google Chrome Canary and Google Chrome Beta are useless. They're not, as it allows issues to be reported and fixed before the actual release. Closing this as it's strongly out of scope and will probably not happen.
